src/main/java/seedu/address/logic/commands/BackCommand.java

```diff
@@ -16,4 +17,15 @@ public CommandResult execute(Model model) {
         return new CommandResult(MESSAGE_BACK_ACKNOWLEDGEMENT, false, false, true);
     }
 
+    /**
+     * Back command can be run on every panel
```

Comment:
I think the Javadoc comment should start with a verb in the present tense according to the 2103 style guide. Maybe phrase it like `Returns true since the Back command can be run on every panel`?

Reply:
Fixed!